convert_hf_to_gguf.py

```diff
@@ -1818,6 +1818,58 @@ def modify_tensors(self, data_torch: Tensor, name: str, bid: int | None) -> Iter
 
         return [(self.map_tensor_name(name), data_torch)]
 
+@Model.register("MiniCPM3ForCausalLM")
```

Comment:
```suggestion

@Model.register("MiniCPM3ForCausalLM")
```

Lint is [current breaking](https://github.com/ggerganov/llama.cpp/actions/runs/10849273173/job/30108206656?pr=9322#step:4:77) on this -- need to add an additional blank line above your class definition.

Reply:
Thank you for your comments.